### PR TITLE
New package: oxen-io.lokinet version 0.9.11

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/132447)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please recheck these Pull Request carefully before merging. If it is a incorrect-removing Pull Request and the URLs don't have any issue while manual checking, please close this Pull Request directly(It's best to let Exorcism0666 know).

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
image

**Comment by stephengillie** [comment]
Hi @Exorcism0666,

The installation isn't unattended. Is there an installer switch to have the package install silently?

(Automated response - build 522.)

**Comment by Exorcism0666** [comment]
@stephengillie done!

**Comment by OfficialEsco** [comment]
The community have created a few tools in https://github.com/microsoft/winget-pkgs/tree/master/Tools
I've not used this in a loong time, but [SandboxTest.ps1](https://github.com/microsoft/winget-pkgs/blob/master/Tools/SandboxTest.ps1) should be one you want to use :)
[PRTest.ps1](https://github.com/microsoft/winget-pkgs/blob/master/Tools/PRTest.ps1) if you have a PR already
(You need winget-pkgs cloned to your machine for these scripts)

File: manifests/o/oxen-io/lokinet/0.9.11/oxen-io.lokinet.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: oxen-io.lokinet
+PackageVersion: 0.9.11
+PackageLocale: en-US
+Publisher: oxen-io
```
Comment (OfficialEsco) [inline manifests/o/oxen-io/lokinet/0.9.11/oxen-io.lokinet.locale.en-US.yaml:7]:
```suggestion
Publisher: lokinet.org
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
> Name          : Lokinet
> Publisher     : lokinet.org
> Version       : 0.9.11
> ProductCode   : Lokinet
> Registry Path : Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\Lokinet
> Uninstall     : winget uninstall --id "Lokinet"

**Comment by OfficialEsco** [comment]
Just realized i answered the question from https://github.com/microsoft/winget-pkgs/pull/132532#issuecomment-1877776567 :D

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**